test/models/music_test.rb

```diff
@@ -0,0 +1,32 @@
+require 'test_helper'
+
+class MusicTest < ActiveSupport::TestCase
+  describe "Music" do
+    before do
+      @music_track = Music.spotify_find("track", "0fDFzVTG8c2fW9EM5f1RHM")
+      @music_album = Music.spotify_find("album", "1OYLLmDS0pJVFRbUo19vrG")
+      @music_artist = Music.spotify_find("artist", "0cOpQTKJdCEzz2wplnnyP6")
+    end
+
+    it "Knows various track attributes", :vcr do
+      assert_equal @music_track.name, "Indian Food"
+      assert_equal @music_track.type, "track"
+      assert_equal @music_track.uri, "spotify:track:0fDFzVTG8c2fW9EM5f1RHM"
+      assert_equal @music_track.external_urls["spotify"], "https://open.spotify.com/track/0fDFzVTG8c2fW9EM5f1RHM"
+    end
+
+    it "Knows various album attributes", :vcr do
+      assert_equal @music_album.name, "Thunderbox"
+      assert_equal @music_album.type, "album"
+      assert_equal @music_album.uri, "spotify:album:1OYLLmDS0pJVFRbUo19vrG"
+      assert_equal @music_album.external_urls["spotify"], "https://open.spotify.com/album/1OYLLmDS0pJVFRbUo19vrG"
+    end
+
+    it "Knows various artist attributes", :vcr do
+      assert_equal @music_artist.name, "Miso Banana Trio"
+      assert_equal @music_artist.type, "artist"
+      assert_equal @music_artist.uri, "spotify:artist:0cOpQTKJdCEzz2wplnnyP6"
+      assert_equal @music_artist.external_urls["spotify"], "https://open.spotify.com/artist/0cOpQTKJdCEzz2wplnnyP6"
+    end
+  end
+end
```

Comment:
Thanks for looking this over.  I was trying to be better about deleting
commented out code.  I think I was still thinking I might use it, but I
will do a last sweep over because you are right, it looks better if it's
not in there.  I was unaware that the data should have been in the model
instead of the controller.  I don't feel like anyone ever said to me these
belong in the model and I'm so use to the controller.  I am definitely
aware of that now and will figure out where my methods should actually go.
Thanks so much for you feedback, it's much appreciated as I want to get
better!

Adriana

On Tue, May 24, 2016 at 3:33 PM, Lauren G <notifications@github.com> wrote:

> In test/models/music_test.rb
> <https://github.com/Ada-C5/tunes-takeout/pull/8#discussion_r64486934>:
>
> > +
> > +    it "Knows various album attributes", :vcr do
> > +      assert_equal @music_album.name, "Thunderbox"
> > +      assert_equal @music_album.type, "album"
> > +      assert_equal @music_album.uri, "spotify:album:1OYLLmDS0pJVFRbUo19vrG"
> > +      assert_equal @music_album.external_urls["spotify"], "https://open.spotify.com/album/1OYLLmDS0pJVFRbUo19vrG"
> > +    end
> > +
> > +    it "Knows various artist attributes", :vcr do
> > +      assert_equal @music_artist.name, "Miso Banana Trio"
> > +      assert_equal @music_artist.type, "artist"
> > +      assert_equal @music_artist.uri, "spotify:artist:0cOpQTKJdCEzz2wplnnyP6"
> > +      assert_equal @music_artist.external_urls["spotify"], "https://open.spotify.com/artist/0cOpQTKJdCEzz2wplnnyP6"
> > +    end
> > +  end
> > +end
>
> testing! yay!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Ada-C5/tunes-takeout/pull/8/files/3a6a31a47083c305cb9289b86b2cb7f8833231d2#r64486934>
>